.github/workflows/unittest.yml

```diff
@@ -23,8 +23,11 @@ jobs:
         run: |
           pip install tensorflow==${{matrix.tf-version}}
           pip install -e .[test]
-      - name: Test with pytest
-        run: pytest . -n auto --cov=larq_zoo --cov-report=xml --cov-config=.coveragerc
+      - name: Test data and models with pytest
+        run: pytest tests/{data_test.py,models_test.py,utils_test.py} -n auto --cov=larq_zoo --cov-report=xml --cov-config=.coveragerc
```

Comment:
To make it easier to add new tests, should we use this instead?
```suggestion
        run: pytest . -n auto --ignore=tests/train_test.py --cov=larq_zoo --cov-report=xml --cov-config=.coveragerc
```

Reply:
Yep makes sense